dk-installer.py

```diff
@@ -1431,9 +1452,12 @@ class TestGenPullImagesStep(Step):
 
     def execute(self, action, args):
         remaining_attemps = TESTGEN_PULL_RETRIES
+        docker_compose_file_path = get_docker_compose_path()
         while True:
             try:
-                with action.start_cmd("docker", "compose", "pull", "--policy", "always") as (proc, stdout, stderr):
+                with action.start_cmd("docker", "compose", "-f", docker_compose_file_path, "pull") as (
```

Comment:
if I understood the logic correct, this will crash on linux, because `get_docker_compose_path()` would return `None`